services/ui-src/src/components/pages/Dashboard/DashboardPage.tsx

```diff
@@ -168,6 +169,9 @@ export const DashboardPage = ({ reportType }: Props) => {
       await fetchReportsByState(reportType, activeState);
       setReportId(undefined);
       setReleasing(false);
+
+      //use discourse to open modal
```

Comment:
nit: for the comment on line 173 should it be //useDisclosure to open modal?

Reply:
fixed!